layers/+lang/haskell/packages.el

```diff
@@ -54,6 +55,13 @@
   (use-package ghc
     :defer t))
 
+(defun haskell/init-dante ()
+  (use-package dante
+    :defer t
+    :commands 'dante-mode)
```

Comment:
`dante-mode` should not be quoted, but actually this line is not necessary, as `dante-mode` [is autoloaded](https://github.com/jyp/dante/blob/master/dante.el#L157-L158).